task_sdk/src/airflow/sdk/definitions/xcom_arg.py

```diff
@@ -343,6 +343,7 @@ def resolve(self, context: Mapping[str, Any]) -> Any:
             task_ids=task_id,
             key=self.key,
             default=NOTSET,
+            map_indexes=None,
```

Comment:
I still have a nasty feeling that there is some edge case where we need to pass a map index here, but I can't think exactly what it is, so this is right for now.

Reply:
Hmmm i gave it a fair thought but couldnt really think about a case too. I guess we handle as it comes 🗡️